files/lab/workshop/content/cloning.md

```diff
@@ -349,12 +349,6 @@ Then enable nginx service:
 systemctl daemon-reload && systemctl enable --now nginx
 ```
 
-Which should show:
```

Comment:
why removing this ? bit concerned if it doesn't show, means nginx might not be correctly enabled ?

Reply:
Deleted by mistake. Added again in the last commit.